CHANGELOG.md

```diff
@@ -26,6 +26,12 @@ In preparation for stabilizing HTTP library instrumentation soon:
 - `*KtorClientTracing*` and `*KtorServerTracing*` have been deprecated and renamed to
   `*KtorClientTelemetry*` and `*KtorServerTelemetry*`
   ([#12855](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12855))
+- Some Java agent instrumentation suppression keys have been renamed to match their module names:
+  - `elasticsearch-rest-6.0` --> `elasticsearch-rest-6.4`
+  - `twitter-util-core` --> `finagle-http`
```

Comment:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/finagle-http-23.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/finaglehttp/v23_11/TwitterUtilCoreInstrumentationModule.java
Instruments classes in `com.twitter:util-core` that finagle depends on, finatra also uses the same dependency but the tests don't seem to require this instrumentation. Maybe the instrumentation should be moved to a separate module? Though this could still be a bit weird as it is tested in the finagle module.

Reply:
> Instruments classes in `com.twitter:util-core` that finagle depends on

Thanks, I missed this, I kept the `twitter-util-core` suppression key (and just added the normal finagle suppression keys)